### DISCUSSION TITLE
数据库再down的状态下，程序启动时连接，设置连接池依然成功，实际操作数据库的时候才报错‘"Error occurred while creating a new object: Connection refused (os error 111)’

### DISCUSSION THREAD

**Discussion by nan19890108** [rbatis/rbatis#528] [Q&A]
使用版本：
rbatis = { version = "4.5"}
rbdc-pg = { version = "4.5"}
rbdc-mssql = {version="4.5"}
rbs = { version = "4.5" }
rbdc-pool-deadpool = "4.5.7"

**Comment by zhuxiujia** [comment]
你用link函数的话，就是初始化驱动并连接数据库校验能否成功了。
